### PR TITLE
Complete typescript-operators

### PR DESCRIPTION
**Pull request by nmotamedi**
Screen Shot 2024-02-27 at 10 59 09 AM

### CODE REVIEW

File: typescript-operators/main.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+const v4 = [3, 5, 7, 9];
+const v5: Customer = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline typescript-operators/main.ts:27]: This is technically still firing for both cases. This should be written in a way that the `console.log` will only fire if `v1` or `v2` evaluate to `truthy`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Nader!

**Comment by nmotamedi** [comment]
Screen Shot 2024-02-27 at 11 29 51 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**